### PR TITLE
Add descriptions for each item in limits and triggers table

### PR DESCRIPTION
**Pull request by JameStark**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [ ] Changes have been done against dev branch, and PR does not conflict
- [ ] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/889)

### CODE REVIEW

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 1]
```diff
@@ -41,6 +44,22 @@ The gas limit for Starknet blocks is therefore inherited from the gas limit for
...
+## Max transaction size (Cairo steps)
+The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.
+
+This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.
+
+## Max contract bytecode size / Number of felts in the program
+
+The maximum size of the bytecode or program that a smart contract can have on Starknet.
+
+Bytecode is the low-level code that smart contracts are composed of. Limiting this size helps manage the complexity of contracts and the overall efficiency of the network.
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:56]:
```suggestion
Bytecode is the low-level code that comprises smart contracts. Limiting this size helps manage the complexity of contracts and the overall efficiency of the network.
```

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 2]
```diff
@@ -41,6 +44,22 @@ The gas limit for Starknet blocks is therefore inherited from the gas limit for
 The gas limit for Starknet blocks is inherited from the gas limit for Ethereum blocks.
 ====
 
+## Max transaction size (Cairo steps)
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:47]: Change all # headings to AsciiDoc = headings.

**Review by stoobie — CHANGES_REQUESTED** [review]
2 Minor comments.

One major comment: It doesn't make sense for all these sections under the table. Just put these descriptions directly in the table under the term.

For example:

```
| *Max transaction size (Cairo steps)*

The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.

This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.
```

So this table cell should appear like this (or course it'll be wrapped):

-----

**Max transaction size (Cairo steps)**

The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.

This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc (hunk 1 above)
Comment (JameStark) [reply components/Starknet/modules/tools/pages/limits_and_triggers.adoc:56]: Done

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc (hunk 2 above)
Comment (JameStark) [reply components/Starknet/modules/tools/pages/limits_and_triggers.adoc:47]: Done

**Review by stoobie — APPROVED** [review]
LGTM

**Pull request merged**